int32 can_id
int8 can_dlc
char[] can_data
---
int8 status
